sdk/search/Azure.Search.Documents/samples/Sample06_EncryptedIndex.md

```diff
@@ -0,0 +1,126 @@
+# Azure.Search.Documents Samples - Customer-managed Encryption Keys
+
+Azure Cognitive Search automatically encrypts indexed content at rest with [service-managed keys](https://docs.microsoft.com/azure/security/fundamentals/encryption-atrest#azure-encryption-at-rest-components). If more protection is needed, you can supplement default encryption with an additional encryption layer using keys that you create and manage in Azure Key Vault. This sample shows you how to set up and use customer-managed keys to further encrypt your data source connections, skillsets, indexers, and indexes for additional security.
+
+## Getting started
+
+To begin using Customer-Managed Keys (CMK), follow steps 1 through 5 to [configure customer-managed keys for data encryption](https://docs.microsoft.com/azure/search/search-security-manage-encryption-keys). This will guide you how to configure your Key Vault, register a service principal if necessary, and grant appropriate access to a service principal for a Key Vault key. This can be a key you created previously, or you can create a key that can wrap and unwrap service keys, including AES and RSA keys like in the following example:
+
+```bash
+# Create a resource group for your Key Vault and Cognitive Search service.
+# At this time, CMK is limited to certain regions including West US 2,
+# so we'll create everything in that location.
+az group create -n "<group-name>" -l westus2
+
+az keyvault create -g "<group-name>" -n "<vault-name>" -l westus2 --enable-purge-protection
+
+# Create a service principal for RBAC, or you can use Managed Identity.
+az ad sp create-for-rbac -n "<app-name>" --skip-assignment
+```
+
+The last command will provide output like the following:
+
+```text
```

Comment:
Probably should be json instead of text.

Reply:
Seems GH-flavored markdown supports it, but last I tried docs.microsoft.com does it and we do mean to target both. That said, it certainly couldn't hurt.